Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -1794,3 +1799,5 @@ def main():
 
 if __name__ == 'builtins':
     main()
+
+register_module_line('Microsoft Teams', 'end', __line__())
```

Comment:
```suggestion
```
Please delete this line.

Reply:
This was part of the original integration. It was not added by us.

Reply:
I know, it's because you worked via the UI and it's a bug we're trying to fix right now.
Please remove this line.